Src/AutoNSubstitute/AutoNSubstitute.csproj

```diff
@@ -64,6 +64,7 @@
   <ItemGroup>
     <Compile Include="AutoConfiguredNSubstituteCustomization.cs" />
     <Compile Include="AutoNSubstituteCustomization.cs" />
+    <Compile Include="AutoSubstituteCustomization.cs" />

```

Comment:
I don't have a strong preference on either of these points, simply trying to get to an agreement. Since you don't seem to like the idea of consistently applying the `N` prefix consistently, how about this: 

- move code from the new `AutoSubstituteCustomization` class into the two existing  `AutoNSubstituteCustomization` and `AutoConfiguredNSubstituteCustomization` classes;
- eliminate the new `AutoSubistituteCustomization` class;
- rename the new `SubstituteBuilder` to `SubstituteRequestHandler` to avoid confusion with the existing `NSubstituteBuilder`